app/models/requests/PrevalidateTraderRequest.scala

```diff
@@ -18,7 +18,7 @@ package models.requests
 
 import play.api.libs.json.{Format, Json}
 
-case class PrevalidateTraderRequest(ern: String, productCodes: Seq[String])
+case class PrevalidateTraderRequest(ern: String, entityGroup: String, productCodes: Seq[String])
```

Comment:
Could entityGroup be made more rigid by making it have EntityGroup type? I assume the JSON parser can handle converting that into the correct string?

Reply:
Updated at https://github.com/hmrc/emcs-tfe-frontend/pull/169/commits/85030186d2e1a969fbc1ff221d8201450579c0ff